js/lib/rich-text/parser.js

```diff
@@ -0,0 +1,6 @@
+export default function(content) {
+
+    const buffer = document.createRange();
```

Comment:
Need to read about these as I am not familiar with them...

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Range